### PR TITLE
Ensure that mode.getFullVersion_ falls back to the full version and not just major

### PR DESCRIPTION
**Pull request by dvoytenko**
Partial for #5940.

### CODE REVIEW

**Comment by erwinmombay** [comment]
@jridgewell for final review. not sure if this affect sw in any way

File: src/mode.js [hunk 1]
```diff
@@ -173,5 +173,10 @@ export function getFullVersion_(win, isLocalDev) {
     return win.AMP_CONFIG.v;
   }
 
-  return version;
+  // Currently `$internalRuntimeVersion$` and thus `mode.version` contain only
+  // major version. The full version however must also carry the minor version.
+  // We will default to production default `01` minor version for now.
+  // TODO(erwinmombay): decide whether $internalRuntimeVersion$ should contain
+  // minor version.
+  return `01${version}`;
```
Comment (jridgewell) [inline src/mode.js]: `01` may not always be the Prod prefix, though. I think. @cramforce would know better than I would.

Comment (dvoytenko) [inline src/mode.js]: Yes, `01` is not always the best. But it should always be there. And we have no better information for a fallback at this time. Separately, I asked @erwinmombay to consider to simply update `$internalRuntimeVersion$` to a full version or add another replacement like this for minor version.

Comment (jridgewell) [inline src/mode.js]: I took to using two types, `AmpVersion` and `RtvVersion` in the SW to help catch issues where I used a non-prefixed value in a place that requires a prefix. Maybe we should do the same here to catch these kinds of issues?

Comment (dvoytenko) [inline src/mode.js]: Something like `rtvVersion` definitely makes sense.

Comment (dvoytenko) [inline src/mode.js]: Ok. I added `rtvVersion` field to disambiguate the two.

**Comment by jridgewell** [comment]
LGTM.

**Pull request merged**